db/seeds/communities.yml

```diff
@@ -0,0 +1,5 @@
+- id: 1
+  name: Test
+  host: localhost
```

Comment:
These two lines are all you need here. ID, `created_at`, and `updated_at` are all handled for you by Rails.